docs/src/tutorial-first-steps.md

```diff
@@ -4,10 +4,10 @@ Kani is unlike the testing tools you may already be familiar with.
 Much of testing is concerned with thinking of new corner cases that need to be covered.
 With Kani, all the corner cases are covered from the start, and the new concern is narrowing down the scope to something manageable for the checker.
 
-Consider this first program (which can be found under [`kani-first-steps`](https://github.com/model-checking/kani/tree/main/docs/src/tutorial/kani-first-steps/)):
+Consider this first program (which can be found under [`first-steps-original`](https://github.com/model-checking/kani/tree/main/docs/src/tutorial/first-steps-original/)):
```

Comment:
Is there any other naming convention that we can use over original vs final? Maybe before vs after or initial vs final?

Reply:
Yes, we could use "initial" and "final" or version numbers if you like (e.g., `first-steps-v1` and `first-steps-v2`). I don't like "before" and "after" because they remind me of TV commercials 😆

Reply:
Sure let's not do before/after 😆 

I personally like v1 and v2, if you agree with it as well, i think it makes sense to rename it to that.

Reply:
Updated to use `first-steps-v1` and `first-steps-v2`!